### PR TITLE
Parallel corpus improvements

### PR DESCRIPTION
**Pull request by ArbinTimilsina**
This PR consists of few improvements to the parallel corpus, mainly:
- Indonesian and Korean parallel dataset are added
- Cleanup is performed using language identification
- Tests are added

Skipping a line if the text on either side contains different numbers of '♪' reduces the corpus size by less than a percent and skipping if languages on either side are not identified with confidence reduces it by 3 to 16%.

### CODE REVIEW

File: tests/test_parallel_corpus.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    # Should be 26 lower case alphabets + '▁'
+    out_file.seek(0)
+    count = len(out_file.readlines())
+    assert count == 27
+
+
+@pytest.mark.parametrize(
+    'text, expected',
+    [
+        (DECODED[0], ENCODED[0]),
```
Comment (rspeer) [inline tests/test_parallel_corpus.py]: Can all of this be replaced with `zip(DECODED, ENCODED)`?

File: tests/test_parallel_corpus.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+from exquisite_corpus.parallel_corpus import (
+    cleanup_parallel_file, decode_pieces_with_sp, encode_with_sp_as_pieces,
+    get_vocabulary_from_sp, train_sentencepiece
+)
+
+DECODED = [
+    'abcdefghijklmnopqrstuvwxyz',
+    'this is a sample text',
+    ''
```
Comment (rspeer) [inline tests/test_parallel_corpus.py]: Why is there a third item here that doesn't correspond to anything in ENCODED, and seems to not be used? Am I missing where it's used?

File: tests/test_parallel_corpus.py (hunk 1 above)
Comment (ArbinTimilsina) [reply tests/test_parallel_corpus.py]: Yes, it can be.

File: tests/test_parallel_corpus.py (hunk 2 above)
Comment (ArbinTimilsina) [reply tests/test_parallel_corpus.py]: Ah, I think it is a leftover- I will remove it.

**Pull request merged**